frontend/frontend/src/components/Game.vue

```diff
@@ -33,123 +34,18 @@ export default {
 		Match,
 		Timer,
 	},
+	mounted() {
+		axios.get("/api/MetAPI").then((response) => {
+			response.data.forEach((card) => {
+				this.cards.push(card);
+			});
+			this.cards=response.data;
```

Comment:
```suggestion
```

Reply:
I'm not sure I understand this suggestion Abi - why would we remove this line?

Reply:
The above for loop adds the cards to data variable... they both work so we only need one :) it can be either the foreach loop or the assertion